src/app/components/index.ts

```diff
@@ -1 +1,3 @@
 //export * from './warehouse'
+export * from './homepage'
+export * from './gamepage'
```

Comment:
I think components should be smaller things, for screens I would create new folder, e.g. screens and put them there